src/views/tables/FillsTable.tsx

```diff
@@ -197,6 +201,26 @@ const getFillsTableColumnDef = ({
           </TableCell>
         ),
       },
+      [FillsTableColumnKey.Total]: {
+        columnKey: 'total',
+        getCellValue: (row) => MustBigNumber(row.price).times(row.size).toNumber(),
```

Comment:
another random thought - we should probably standardize around doing math in big number or not, kinda awkward to pass around though